policy/modules/contrib/wireguard.te

```diff
@@ -0,0 +1,44 @@
+policy_module(wireguard, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+attribute_role wg_roles;
```

Comment:
Is using roles necessary for some of the tested scenarios?

Reply:
no necessary